index.html

```diff
@@ -9,7 +9,7 @@
          defer
          ></script>
      <script class="remove"
-             src="https://cdn.jsdelivr.net/gh/digitalbazaar/respec-vc@2.0.1/dist/main.js"></script>
+             src="https://cdn.jsdelivr.net/gh/transmute-industries/respec-vc-jwt@0.0.9/dist/main.js"></script>
```

Comment:
`vc-jwt` -> `vc-jose-cose` ?

Reply:
I'm going to resolve these comments, the plugin was built before the rename, and it won't be updated until after the work item makes progress on media types.